kuksa_databroker/databroker/src/main.rs

```diff
@@ -105,6 +106,22 @@ fn init_logging() {
     info!("{}", output);
 }
 
+async fn apply_default(id: i32, val: &serde_json::Value, broker : &DataBroker) {
```

Comment:
This function would need to get _a lot_ longer, getting access to metadata and creating the correct "DataValue"

Reply:
Moved the conversion `serde_json::Value` -> `DataValue` directly into the parser instead.